tests/e2e/dsc_creation_test.go

```diff
@@ -343,7 +389,7 @@ func (tc *testContext) testUpdateComponentReconcile() error {
 				Namespace: testDeployment.Namespace,
 			},
 			Spec: autoscalingv1.ScaleSpec{
-				Replicas: 3,
+				Replicas: 4, // rhoai has 5 pods
```

Comment:
But if it is patched to 3, should work as well, shouldn't?

Reply:
yes, it should work by either 3 or 4.
only think  from 5->4->5 should be faster than 5->3->5 if the logic is to test reconcile works or not.